kotlinx.fuzz.junit/src/main/kotlin/kotlinx/fuzz/junit/KotlinxFuzzJunitEngine.kt

```diff
@@ -155,23 +177,28 @@ class KotlinxFuzzJunitEngine : TestEngine {
                 engineDescriptor,
                 config,
                 isRegression,
+                supportJazzerTargets = config.global.supportJazzerTargets,
             ),
         )
     }
 
     companion object {
-        val isKFuzzTestContainer: (Class<*>) -> Boolean = { klass ->
+        private fun isKFuzzTestContainer(klass: Class<*>, supportJazzerTargets: Boolean): Boolean =
             ReflectionSupport.findMethods(
                 klass,
-                { method: Method -> method.isFuzzTarget() },
+                { method: Method -> method.isFuzzTarget(supportJazzerTargets) },
                 HierarchyTraversalMode.TOP_DOWN,
             ).isNotEmpty()
-        }
 
-        private fun Method.isFuzzTarget(): Boolean =
+        internal fun Method.isFuzzTarget(supportJazzerApi: Boolean): Boolean =
             AnnotationSupport.isAnnotated(this, KFuzzTest::class.java) &&
-                parameters.size == 1 &&
-                parameters[0].type == KFuzzer::class.java
+                parameterCount == 1 &&
+                parameters[0].type == KFuzzer::class.java ||
+                (supportJazzerApi && isJazzerFuzzTarget())
+
+        private fun Method.isJazzerFuzzTarget(): Boolean =
+            AnnotationSupport.isAnnotated(this, FuzzTest::class.java) && parameterCount == 1 &&
+                (parameters[0].type == ByteArray::class.java || parameters[0].type == FuzzedDataProvider::class.java)
```

Comment:
Why are we ignoring jazzer's autofuzz?

Reply:
1. I didn't think about it
2. I think it's unimportant
3. I'll add a warning, that autofuzz in unsupported